backend/.gitignore

```diff
@@ -35,3 +35,4 @@
 /config/master.key
 /env
 /env.*
+/.env
```

Comment:
And what is it? Why you moved character "." in start?

Reply:
This file has been added to gitignor because it contains important hidden variables that should not be shared. The dot at the beginning of the file name is the full name of the ".ENV" file